### ISSUE TITLE
Openshift Task creation fails with `an empty namespace may not be set during creation`

### ISSUE THREAD

**Issue by Schnitzel** [amazeeio/lagoon-kbd#67]
I merged #25 and released it under amazeeio/lagoon-builddeploy:v0.1.14-pr25, but we have now an issue when creating a task via the controller on an openshift:

- controller is running in`lagoon-build-deploy` on ch1
- deploys to `drupal-example-1` also on ch1

```
2021-06-28T20:19:02.575Z	INFO	controllers.LagoonTask	Checking task pod for: lagoon-task-10692-ngpw1d	{"lagoontask": "drupal-example-1-master/lagoon-task-10692-ngpw1d"}
2021-06-28T20:19:02.587Z	INFO	controllers.LagoonTask	Creating task pod for: lagoon-task-10692-ngpw1d	{"lagoontask": "drupal-example-1-master/lagoon-task-10692-ngpw1d"}
2021-06-28T20:19:02.587Z	INFO	controllers.LagoonTask	Unable to create task pod for project drupal-example-1, environment master: an empty namespace may not be set during creation	{"lagoontask": "drupal-example-1-master/lagoon-task-10692-ngpw1d"}
2021-06-28T20:19:02.587Z	DEBUG	controller-runtime.controller	Successfully Reconciled	{"controller": "lagoontask", "request": "drupal-example-1-master/lagoon-task-10692-ngpw1d"}
```

**Comment by shreddedbacon** [comment]
Fixed in #68 and updated ch1 to use amazeeio/lagoon-builddeploy:v0.1.14-pr68 and verified.

Need to write tests for tasks into the controller tests to capture this.

**Comment by Schnitzel** [comment]
confirmed! epic thanks @shreddedbacon
